src/test/resources/formplayer_translatable_strings.txt

```diff
@@ -4,3 +4,6 @@ ethiopian_months=Mäskäräm,T’ïk’ïmt,Hïdar,Tahsas,T’ïr,Yäkatit,Mäga
 nepali_months=Baishakh,Jestha,Ashadh,Shrawan,Bhadra,Ashwin,Kartik,Mangsir,Poush,Magh,Falgun,Chaitra
 form.attachment.invalid=Invalid filetype choosen. Please select a valid multimedia file.
 file.oversize.error.message=Selected file exceeds the file size limit of 3 MB. Please select a file whose size is less than 3 MB.
+form.upload.attachments.threshold.error.message=More than ${0} attachments found for this form.
+form.upload.attachment.invalid=Invalid file ${0} attached with this form.
+form.upload.attachment.oversize=File ${0} is more than the maximum limit of 3 MB.
```

Comment:
is it necessary to duplicate these here? If we remove this file it should pick up the one from `src/main/resources`